src/components/bottom/BottomPanel.svelte

```diff
@@ -23,12 +24,12 @@
   <div class="relative z-1">
     <div
       class="flex items-center justify-between gap-2 pt-4 px-7 m-0 absolute top-0 left-0 right-0">
-      <div class="flex items-center gap-2">
+      <div class="flex items-center gap-4">
         <!-- The live text and info box -->
         <LiveGraphInformationSection />
         <ConnectedLiveGraphButtons />
-        {#if false}
-          <div>
+        {#if $state.reconnectState.reconnecting && $state.isInputConnected}
+          <div class="py-1px bg-white rounded-4xl">
```

Comment:
Only show this for a background/automatic reconnection without introducing new state.